src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -332,6 +355,14 @@ def helm_install_release(chart_path, subscription_id, kubernetes_distro, kuberne
         # Change --timeout format for helm client to understand
         onboarding_timeout = onboarding_timeout + "s"
         cmd_helm_install.extend(["--wait", "--timeout", "{}".format(onboarding_timeout)])
+    if container_registry_repository:
+        cmd_helm_install.extend(["--set", "global.isCustomRegistryEnabled={}".format(True)])
+        cmd_helm_install.extend(["--set", "systemDefaultValues.image.repository={}".format(container_registry_repository)])
+        cmd_helm_install.extend(["--set", "systemDefaultValues.image.releaseName={}".format("azurearck8s")])
```

Comment:
This is set to 'azurearck8s' by defaut in the helm charts, right? Do we really need to set this here?

Reply:
All the production helm charts are in the release name 'azurearck8s' and we will ask customer to put all the images in this path. @shashankbarsin suggested to make this change.